TokboxStreamStats.js

```diff
@@ -0,0 +1,90 @@
+import OpenTok from '@opentok/client'
+
+let timer
+let subscriberStatsByID = {}
+let publisherStatsByID = {}
+
+const videoFrameRate = (stream) => {
+  let frameRate = 30
+
+  return () => {
+    setTimeout(() => stream.getStats((error, stats) => {
```

Comment:
Why is this necessary here?

Reply:
It was a test to make the call `stream.getStats` to perform better because I guess it could be a heavy task. I haven't tested it. Anyways, I should add it to all the calls to `stream.getStats`. Do you remember this video? https://www.youtube.com/watch?v=8aGhZQkoFbQ